roles/ceph-mds/tasks/create_mds_filesystems.yml

```diff
@@ -25,8 +25,7 @@
   changed_when: false
   delegate_to: "{{ groups[mon_group_name][0] }}"
   when:
-    - ceph_release_num[ceph_release] >= ceph_release_num.jewel
-    - ceph_release_num[ceph_release] < ceph_release_num.mimic
+    - ceph_release_num[ceph_release] == ceph_release_num.luminous
```

Comment:
@batrick shouldn't it had been `ceph_release_num[ceph_release] >= ceph_release_num.luminous` ? instead ?

Reply:
No, `allow_multimds` was deprecated in Mimic.